staging/src/kubevirt.io/api/testdata/README.md

```diff
@@ -0,0 +1,155 @@
+# API serialization compatibility tests
+
+This directory tree contains serialized API objects in json and yaml formats.
+
+This creates JSON and YAML files for three APIs exposed by kubevirt in group-version "kubevirt.io/v1", versioned by the release. The main branch is will be stored in `HEAD` directory, previous versions are in `release-0.yy` release directory. While more APIs can be added in the future, the current list includes:
```

Comment:
Is there a plan to extend to other API groups/resources as well (instancetype, snapshot, export, etc)?

Reply:
Yes

Reply:
@mhenriks not in this PR, the plan is to start small, learn if this works for the 3 core APIs. Other APIs could be expanded in the future releases if things go as planned